yarn.lock

```diff
@@ -281,16 +281,6 @@
   dependencies:
     "@parcel/plugin" "2.0.0-rc.0"
 
-"@mozilla/glean@^0.29.0":
```

Comment:
I didn't have yarn installed so I manually removed that. Not sure if a run of yarn might actually remove more because of the deps.